docs/resources/ec_deployment.md

```diff
@@ -151,6 +151,10 @@ The optional `elasticsearch.topology` block supports the following arguments:
 * `size` - (Optional) Amount of memory (RAM) per topology element in the `"<size in GB>g"` notation. When omitted, it defaults to the deployment template value.
 * `size_resource` - (Optional) Type of resource to which the size is assigned. Defaults to `"memory"`.
 * `zone_count` - (Optional) Number of zones the instance type of the Elasticsearch cluster will span. This is used to set or unset HA on an Elasticsearch node type. When omitted, it defaults to the deployment template value.
+* `node_type_data` - (Optional) Node type (data) for the Elasticsearch cluster.
+* `node_type_master` - (Optional) Node type (master) for the Elasticsearch cluster.
```

Comment:
```suggestion
* `node_type_master` - (Optional) Node type for the Elasticsearch cluster (data node).
```

I'm not entirely sure these descriptions provide anything that you can't glean from the argument name, but I would perhaps put the text in brackets at the end to improve readability. Also, the [ES reference](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html#node-roles) refers to node _roles_ rather than types, but it seems our use of node types in descriptions is pretty pervasive to perhaps best to stick with what you have.

Reply:
node roles are a new field which we haven't implemented yet :)